src/main/java/org/team1540/bunnybotTank2023/Robot.java

```diff
@@ -98,6 +98,7 @@ public void teleopInit() {
         if (autonomousCommand != null) {
             autonomousCommand.cancel();
         }
+        robotContainer.shooter.stop();
```

Comment:
why stop shooter on teleop init?

Reply:
Oh that's left over from testing. Since the closed loop setpoints are stored persistently on the Falcons, if I were to set the motor speed and then disable the robot, when I reenable the shooter would immediately spin up to the previous speed, which is not always desirable